lib/attribute-parser.js

```diff
@@ -18,6 +20,9 @@ module.exports = class AttributeParser {
       /^\s+/, // whitespace
       /^[^\s]+/ // word
     ]
+
+    // extend global variables
+    global = Object.assign(global, globalContext)
```

Comment:
es6lint test fail because of 
```
          const context = new vm.createContext(this.globalContext)
```

 **error  A constructor name should not start with a lowercase letter  new-cap**